config/index.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
```

Comment:
Can you rename this file to something more specific like `yarn_search-credential.js` or something in this spirit that actually describes the kind of config you're doing here. Also we might not need a config file for that. It's an implementation detail of the version fetcher IMHO.